packages/ui/components/Chart.vue

```diff
@@ -1,9 +1,13 @@
 <template>
   <div ref="container">
-    <div v-show="tooltip" class="tooltip">
-      <span class="value">{{ formatNumber(value) }}</span>
-      <span class="date"> {{ date }}</span>
-    </div>
+    <ChartTooltip
+      v-show="tooltip"
```

Comment:
`tooltip` never changes so it can be deleted [here](https://github.com/witnet/data-feed-app/pull/104/files#diff-99fc14295a14373a26e1343e634254b67f76828067f0e2419a7c090d10914807R64)
```suggestion
```